app/src/main/java/com/example/cactusnotes/validations/LogInValidator.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.cactusnotes.validations
+
+import com.example.cactusnotes.R
+import com.example.cactusnotes.ValidationResult
+
+class LogInValidator : Validator {
+    override fun validate(field: String): ValidationResult {
+        return when {
```

Comment:
Do you want the recipe for perfection?

Follow this steps ✨:

1. Remove braces from when entries
2. Drop the return keyword and explicit return type and convert function to single-expression function
3. Magic!

![](https://i.pinimg.com/originals/b8/6f/ac/b86fac70a91299d2859e7aa347eb335a.gif)

Reply:
Some magic code for you  :D 
![tenor](https://user-images.githubusercontent.com/34371442/112630969-5f711200-8e47-11eb-9889-054b9cbf8c5f.gif)